__tests__/github-markdown_test.js

```diff
@@ -1,5 +1,5 @@
 const results = {
```

Comment:
Should we convert these tests to `ts` too? I think that'll mean everything is ts!

Reply:
Yep, but perhaps on a following PR.